### ISSUE TITLE
Bug: Graph start to "shake" when changing viewport size

### ISSUE THREAD

**Issue by KingDarBoja** [swimlane/ngx-graph#184]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here
```

**Current behavior**

When changing viewport window size, sometimes the links lines dissapear and the boxes start shaking.

**Expected behavior**

The graph should be displayed without any problem, links shouldn't disappear and text boxes shouldn't shake.

**Reproduction of the problem**

I made a stackblitz project with the same scss styles and component being used on my project. 
[Angular ngx-graph with MDB Boostrap](https://stackblitz.com/edit/angular-ngx-graph-mdb)

**What is the motivation / use case for changing the behavior?**

**Please tell us about your environment:**

From my VS Code:
```
Version: 1.33.0 (system setup)
Commit: 0dd516dd412d42323fc3464531b1c715d51c4c1a
Date: 2019-04-04T15:14:28.026Z
Electron: 3.1.6
Chrome: 66.0.3359.181
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.17134
```

* **ngx-graph version:** 6.0.0-rc.6

* **Angular version:** 7.2.13

* **Browser:** [Chrome Versión 73.0.3683.103]

* **Language:** [TypeScript 3.1.1 or higher]

**Comment by KingDarBoja** [comment]
@marjan-georgiev I think this issue is caused by the transformation matrix not setting on a fixed value due to the default precision (10 decimal numbers after comma). I did a little fix which involves updating the `transformation-matrix` library to `1.15.3` version, which has the new `smoothMatrix` function. This function let us fix the precision number to whatever value we want, so I changed this line:

```ts
updateTransform(): void {
    this.transform = toSVG(this.transformationMatrix);
  }
```
to
```ts
  updateTransform(): void {
    this.transform = toSVG(smoothMatrix(this.transformationMatrix, 100));
  }
``` 

Sadly, the error will still show but only if the parent element containing the graph is too small (like 200px).

More info: [transformation-matrix changelog](https://github.com/chrvadala/transformation-matrix/blob/760585ca50b808d80af051c066fce9632a4ac188/README.md#changelog)
